docs/topics/con_server-sent-events.adoc

```diff
@@ -0,0 +1,16 @@
+[id="server-sent-events_{context}"]
+= Server-sent events
+
+Server-sent events (SSE) is a reactive HTTP-based messaging protocol.
+SSE works by establishing a connection between the event source and the client, and allowing the event source, such as an application server or a messaging broker, to use this connection to push events to the client when they occur.
+The connection remains open after the updates are pushed, and can be re-used to push subsequent updates.
+SSE represents a more resource-efficient alternative to polling, where a new connection must be established each time the client polls the events source for updates.
+As opposed to WebSockets, SSE only pushes events in one direction (that is, from the source to the client).
```

Comment:
SSE only pushes events in one direction--> SSE pushes events only in one direction

Reply:
Using _only_ is tricky:) I agree that my original emphasis is a bit off.
Reworded for maximum emphasis: 

> ...  SSE pushes events in one direction only ...